pkg/ddc/jindo/worker.go

```diff
@@ -214,3 +224,24 @@ func (e *JindoEngine) CheckWorkersReady() (ready bool, err error) {
 
 	return
 }
+
+// getWorkerSelectors gets the selector of the worker
+func (e *JindoEngine) getWorkerSelectors() string {
```

Comment:
add new function for ut ? Will it be used elsewhere ?

Reply:
used by master internal, forget to update